src/features/island/hud/components/settings-menu/sub-settings/DequipBumpkin.tsx

```diff
@@ -0,0 +1,180 @@
+import React, { useEffect, useState } from "react";
+
+import walletIcon from "assets/icons/wallet.png";
+import lockIcon from "assets/skills/lock.png";
+import { SUNNYSIDE } from "assets/sunnyside";
+import { Button } from "components/ui/Button";
+import { interpretTokenUri } from "lib/utils/tokenUriBuilder";
+import { OnChainBumpkin, loadBumpkins } from "lib/blockchain/BumpkinDetails";
+import { wallet } from "lib/blockchain/wallet";
+import { OuterPanel } from "components/ui/Panel";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+import { DynamicNFT } from "features/bumpkins/components/DynamicNFT";
+import { Label } from "components/ui/Label";
+import { useAppTranslation } from "lib/i18n/useAppTranslations";
+import { dequipBumpkin } from "lib/blockchain/Dequipper";
+import { loadWearablesBalanceBatch } from "lib/blockchain/BumpkinItems";
+import { shortAddress } from "lib/utils/shortAddress";
+
+interface Props {
+  onClose: () => void;
+}
+
+export const DequipBumpkin: React.FC<Props> = ({ onClose }) => {
+  const { t } = useAppTranslation();
+
+  const [selectedBumpkinId, setSelectedBumpkinId] = useState<number>();
+
+  const [walletBumpkins, setWalletBumpkins] = useState<OnChainBumpkin[]>();
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  const [showSuccess, setShowSuccess] = useState(false);
+
+  useEffect(() => {
+    const load = async () => {
+      const walletBumpkins = await loadBumpkins(
+        wallet.web3Provider,
+        wallet.myAccount as string
+      );
+
+      setWalletBumpkins(walletBumpkins);
+
+      setIsLoading(false);
+    };
+
+    load();
+  }, []);
+
+  const dequip = async () => {
+    const bumpkin = (walletBumpkins ?? []).find(
+      (b) => Number(b.tokenId) === selectedBumpkinId
+    );
+
+    if (!bumpkin) {
+      return;
+    }
+
+    setIsLoading(true);
+
+    // Get all the IDs + Amounts currently on the Bumpkin
+    const wearables = await loadWearablesBalanceBatch(
+      wallet.web3Provider,
+      bumpkin.wardrobe // Bumpkin wallet address
+    );
+
+    await dequipBumpkin({
+      web3: wallet.web3Provider,
+      account: wallet.myAccount as string,
+      bumpkinId: selectedBumpkinId as number,
+      ids: Object.keys(wearables).map(Number),
+      amounts: Object.values(wearables),
+    });
+
+    setShowSuccess(true);
+  };
+
+  if (showSuccess) {
+    return (
+      <div className="p-2">
+        <Label
+          icon={SUNNYSIDE.icons.confirm}
+          type="success"
+          className="my-2 capitalize"
```

Comment:
```suggestion
          className="mb-2 capitalize"
```

Looks like `p-2` is already adding the spacing on the top